### PR TITLE
Fixes golint in runtask_command.go

### PR DESCRIPTION
**Pull request by knrt10**
Signed-off-by: knrt10 <tripathi.kautilya@gmail.com>

**What this PR does / why we need it**:
This PR solves golint errors

**Which issue this PR fixes** : fixes [#1943](https://github.com/openebs/openebs/issues/1943)

### CODE REVIEW

**Comment by prateekpandey14** [comment]
@knrt10 Thanks, can you please rebase with master and resolve conflicts

**Comment by knrt10** [comment]
@prateekpandey14 fixed it.

File: pkg/task/v1alpha1/jiva_volume_delete.go [hunk 1]
```diff
@@ -18,7 +18,8 @@ package v1alpha1
 
 import (
 	"fmt"
-	. "github.com/openebs/maya/pkg/client/http/v1alpha1"
+
+	cl "github.com/openebs/maya/pkg/client/http/v1alpha1"
```
Comment (AmitKumarDas) [inline pkg/task/v1alpha1/jiva_volume_delete.go]: rename `cl` as `rest`

Comment (knrt10) [reply pkg/task/v1alpha1/jiva_volume_delete.go]: ok

Comment (knrt10) [reply pkg/task/v1alpha1/jiva_volume_delete.go]: @AmitKumarDas fixed it

**Comment by knrt10** [comment]
@AmitKumarDas would please tell me why my CI is failing?

File: pkg/task/v1alpha1/runtask_command.go [hunk 2]
```diff
@@ -43,17 +50,21 @@ const (
 type RunCommandCategory string
 
 const (
-	JivaCommandCategory   RunCommandCategory = "jiva"
-	CstorCommandCategory  RunCommandCategory = "cstor"
+	// JivaCommandCategory represents jiva command category
```
Comment (AmitKumarDas) [inline pkg/task/v1alpha1/runtask_command.go]: `JivaCommandCategory categorises the run command as cstor based`

Can you please rephrase all the comments accordingly.

File: pkg/task/v1alpha1/runtask_command.go [hunk 3]
```diff
@@ -19,20 +19,27 @@ package v1alpha1
...
+
+	msg "github.com/openebs/maya/pkg/msg/v1alpha1"
 )
 
 // RunCommandAction determines the kind of action that gets executed by run task
 // command
 type RunCommandAction string
 
 const (
+	// DeleteCommandAction represents delete command action
```
Comment (AmitKumarDas) [inline pkg/task/v1alpha1/runtask_command.go]: `DeleteCommandAction represents a run command as a delete action`

Can you please rephrase the comments for all the actions?

**Comment by knrt10** [comment]
@AmitKumarDas updated, but I have still no idea why CI is failing.

File: pkg/task/v1alpha1/runtask_command.go [hunk 4]
```diff
@@ -168,7 +183,7 @@ type RunCommand struct {
 	Category    RunCommandCategoryList // classification of runtask command
 	Data        RunCommandDataMap      // input data required to execute runtask command
 	SelectPaths []string               // paths whose values will be retrieved after runtask command execution
-	*Msgs                              // store and retrieve info, warns, errors, etc occurred during execution
+	*msg.Msgs                          // store and retrieve info, warns, errors, etc occured during execution
```
Comment (princerachit) [inline pkg/task/v1alpha1/runtask_command.go]: `occurred`

File: pkg/task/v1alpha1/runtask_command.go [hunk 5]
```diff
@@ -400,16 +416,16 @@ type WillRunFn func() bool
...
-	store   ResultStoreFn // mechanism to store runtask command's response
-	willrun WillRunFn     // flag that determines if runtask command will execute or not
-	id      string        // unique identification of the runtask command
-	cmd     *RunCommand   // manages execution of runtask command
-	*Msgs                 // store and retrieve info, warns, errors, etc occurred during execution
+	store     ResultStoreFn // mechanism to store runtask command's response
+	willrun   WillRunFn     // flag that determines if runtask command will execute or not
+	id        string        // unique identification of the runtask command
+	cmd       *RunCommand   // manages execution of runtask command
+	*msg.Msgs               // store and retrieve info, warns, errors, etc occured during execution
```
Comment (princerachit) [inline pkg/task/v1alpha1/runtask_command.go]: `occurred`

**Comment by princerachit** [comment]
@knrt10 please do `make test` that will reveal why your PR is failing ci.

File: pkg/task/v1alpha1/runtask_command.go (hunk 5 above)
Comment (knrt10) [reply pkg/task/v1alpha1/runtask_command.go]: @princerachit you want me to change `msg` to `occurred` ?

Comment (sonasingh46) [reply pkg/task/v1alpha1/runtask_command.go]: No... just cahgne `occured` to `occurred`

Comment (knrt10) [reply pkg/task/v1alpha1/runtask_command.go]: Updated

**Comment by sonasingh46** [comment]
@knrt10 
Thanks for the PR.
Can you please rebase to master or raise another fresh pull request.
Just make sure,prior to changing any variable to fix lint etc .. declare those.( Then you may not get the CI build failures.)

**Pull request closed without merging**